qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,59 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+class TestDirfragKillPoints(TestFragmentation, CephFSTestCase):
```

Comment:
You're double inheriting `CephFSTestCase`.

Reply:
Ack, good catch! That was silly of me. Will push the fix.